test.py

```diff
@@ -75,6 +99,10 @@ def match_properties(actual, expected):
                 if not isinstance(v, exp_v):
                     return False
 
+            elif hasattr(exp_v, '__call__'):
```

Comment:
Oh, cool. It wasn't clear to me from the internet whether that was a Python 3 feature. Fixed in 144c7cc.